phpunit/blocks/renderBlockCorePostExcerpt.php

```diff
@@ -0,0 +1,151 @@
+<?php
+/**
+ * Post Excerpt rendering tests.
+ *
+ * @package WordPress
+ * @subpackage Blocks
+ * @covers ::gutenberg_render_block_core_post_excerpt
+ * @group blocks
+ */
+class Tests_Blocks_RenderBlockCorePostExcerpt extends WP_UnitTestCase {
+
+	/**
+	 * Post object with data.
+	 *
+	 * @var array
+	 */
+	protected static $post;
+
+	/**
+	 * Array of Attributes.
+	 *
+	 * @var int
+	 */
+	protected static $attributes;
+
+	/**
+	 * Setup method.
+	 *
+	 * @param WP_UnitTest_Factory $factory Helper that lets us create fake data.
+	 */
+	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
+
+		self::$post = $factory->post->create_and_get(
+			array(
+				'post_title'   => 'Post Expert block Unit Test',
+				'post_excerpt' => 'Post Expert content',
+			)
+		);
+
+		self::$attributes = array(
+			'moreText'      => '',
+			'excerptLength' => 55,
+		);
+
+		$block = array(
+			'blockName'    => 'core/post-excerpt',
+			'attrs'        => array(
+				'moreText' => '',
+			),
+			'innerBlock'   => array(),
+			'innerContent' => array(),
+			'innerHTML'    => array(),
+		);
+
+		WP_Block_Supports::init();
+		WP_Block_Supports::$block_to_render = $block;
+	}
+
+	/**
+	 * Tear down method.
+	 */
+	public static function wpTearDownAfterClass() {
+		wp_delete_post( self::$post->ID, true );
+	}
+
+	/**
+	 * Test gutenberg_render_block_core_post_excerpt() method
+	 * with empty data.
+	 */
+	public function test_gutenberg_render_block_core_post_excerpt_empty() {
```

Comment:
I suggest using the word `should` in the test method names, as shown in the examples here: https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#test-methods

```suggestion
	public function test_should_render_empty_string_when_excerpt_is_empty() {
```
This is not a hard requirement, but rather a naming convention followed by some committers.
P.S. I think the function name can be omitted since this test class only tests one function.